addons/mail/models/mail_mail.py

```diff
@@ -165,20 +165,6 @@ def unlink(self):
             self.env['mail.message'].browse(mail_msg_cascade_ids).unlink()
         return res
 
-    @api.model
-    def _add_inherited_fields(self):
```

Comment:
Why removing this ? Is it not relevant anymore ? Don't see anything in commit about that part.

Reply:
Up ? @rco-odoo